content/es/guides/directory-structure/pages.md

```diff
@@ -0,0 +1,359 @@
+---
+title: páginas
+description: El directorio de `pages` (páginas) contiene las vistas y rutas de la aplicación. Nuxt.js se encarga de leer todos los archivos de `.vue` dentro de este directorio, y automáticamente crea la configuración de las rutas por ti.
+position: 10
+category: directory-structure
+csb_link: https://codesandbox.io/embed/github/nuxt-academy/guides-examples/tree/master/04_directory_structure/11_pages?fontsize=14&hidenavigation=1&theme=dark
+questions:
+  - question: ¿En cual directorio puedes poner un componente de página?
+    answers:
+      - views
+      - pages
+      - vues
+    correctAnswer: pages
+  - question: Para crear rutas manualmente, necesitas configurar el archivo router.js.
+    answers:
+      - cierto
+      - falso
+    correctAnswer:
+  - question: Puedes crear rutas de archivos con la extensión `.js` o `.ts`.
+    answers:
+      - cierto
+      - falso
+    correctAnswer: cierto
+  - question: ¿Cuándo se llama la propiedad asyncData?
+    answers:
+      - Antes de cargar el componente
+      - Mientras se carga el componente
+      - Luego de cargar el componente
+    correctAnswer: Antes de cargar el componente
+  - question: ¿En qué propiedad se añaden los meta tags?
+    answers:
+      - head
+      - meta
+      - metaTags
+    correctAnswer: head
+  - question: Cuál propiedad se usa para añadir un layout diferente a una página?
```

Comment:
Here I would add an opening question mark ( ¿ ) at the beginning of the sentence.

Reply:
Man, i just noticed that, im missing them in some of the files. Thanks!

- [ ] fix